### ISSUE TITLE
Emaged doors electronics can be fixed

### ISSUE THREAD

**Issue by Zealith-Gamer** [space-wizards/space-station-14#26882]
## Description

Anyone can fix and emaged door by removeing its electronics, changing the access and putting it back in.

**Reproduction**

-Emag door
-deconstruct door till you get door electronics
-Use multitool (used hand in video, different bug)
-change access
-Put the board back into door and build door back up.

**Video**

https://youtu.be/CJZrD8lgUi4

**Additional context**

Yes, i know you dont have to open the panel and all that.

**Comment by Ghagliiarghii** [comment]
Seems to me like the door was emagged, you disassembled it so the emagged door is now gone and instead you have a door assembly, completely new thing btw, then reassembled a new door (not the original emagged door) with the assembly, which, since it's a new door, wasn't emagged. Intended behavior.

**Comment by Zealith-Gamer** [comment]
no, the door electronic isn't ment to be fixable. That's what gets fried by the emag. You are ment to go get a new door electronic. The rest of the door is irrelevent
